src/documentation/reference/ReferenceToolkit.tsx

```diff
@@ -74,7 +74,7 @@ const ActiveToolkitLevel = ({
         <List flex="1 1 auto">
           {(module.children ?? []).map((child) => (
             <ListItem key={child.id}>
-              <ReferenceNode docs={child} width="100%" anchor={anchor} mb={0} />
```

Comment:
I'm convinced we don't need `mb={0}` here unless you can find somewhere it is used? I can't see it myself.